data/scenarios/Tutorials/give.yaml

```
@@ -3,32 +3,59 @@ name: Give
 description: |
   Learn how to give items to other robots.
 objectives:
-  - goal:
+  - id: give_bit
+    teaser: Give base bit
+    goal:
       - |
         Robots can transfer items between them using the `give`
         command. This command takes two arguments: the target
         robot, and the name of the item to give.  The target robot
         is often the special `base` robot, but could be any robot.
-      - In this scenario, you will need to collect the items scattered around your `base`, by `build`ing one or more robots to collect them and then `give` them to you.
+      - |
+        In this scenario, you will need to collect the items scattered
+        around your `base`, by `build`ing one or more robots to collect
+        them and then `give` them to you.
+      - |
+        This is a training simulation, so the items will reappear after a while.
+      - |
+        Start with `build`ing a robot to `grab` the `bit (0)`{=entity} east of the base.
```

```suggestion
        Start with `build`ing a robot to `grab` the `bit (0)`{=entity} east of the base, then return to `give` it to the `base`.
```
I know we have just been explicitly talking about `give`, but I think it's worth repeating the instruction here.  The first time I tried this scenario I read "start by building a robot to grab the bit" and I figured turning around and giving it to the base would come later.  So I did literally what it said, I built a robot to go and `grab` the bit and... nothing happened.